churn

```diff
@@ -5,17 +5,7 @@ require_once(__DIR__ . '/bootstrap.php');
 
 use Churn\Commands\ChurnCommand;
 use Symfony\Component\Console\Application;
-use Churn\Configuration\Config;
-use Churn\Managers\FileManager;
-use Churn\Factories\ProcessFactory;
-use Churn\Results\ResultsParser;
-use Symfony\Component\Yaml\Yaml;
-
-$config = Config::create(Yaml::parse(@file_get_contents(getcwd() . '/churn.yml')) ?? []);
-$fileManager = new FileManager($config->getFileExtensions(), $config->getFilesToIgnore());
-$processFactory = new ProcessFactory($config);
-$resultsParser = new ResultsParser;
 
 $application = new Application();
-$application->add(new ChurnCommand($config, $fileManager, $processFactory, $resultsParser));
+$application->add(new ChurnCommand);
```

Comment:
I'd rather leave everything injected into the `ChurnCommand` class if possible. I just recently made the change to have its dependencies injected. So only want to revert if there's a really good reason why. Is it possible to go back to having them injected?

Reply:
I tried that at first the issue is that the `$config` variable can only be created once the it's known if the default or alternative configuration is used. Since the other classes depend on `$config` variable being set makes it not easy to do open to suggestions to fix this though

Reply:
Ah ok yeah you are right. I think the `ChurnCommand` is do for a refactoring anyways so I'll probably redo this bit in the near future.

If you are able to get the build to pass though that would be great. Let me know if you need a hand with it.

Reply:
Issue is conflicting php-cs rule (i think). I'll fix it in an hour or two by removing the method.

I was thinking about better structure for the ChurnCommand last night. Some suggestions was to pull the business logic and presentation / input logic in separate objects.

Reply:
@bmitch  build is passing now